rest/model/task.go

```diff
@@ -148,6 +152,13 @@ type APIAbortInfo struct {
 	PRClosed   bool   `json:"pr_closed,omitempty"`
 }
 
+type APITaskCost struct {
+	// OnDemandCost is the cost calculated using only on-demand rates
+	OnDemandCost *float64 `json:"on_demand_cost,omitempty"`
+	// AdjustedCost is the cost calculated using the finance formula with savings plan and on-demand components
+	AdjustedCost *float64 `json:"adjusted_cost,omitempty"`
```

Comment:
Are these in cents or dollars? It might be worth adding the currency to the name or to the comment if you'd like

Reply:
I think we can leave this out for now and add more docs later